UnofficialSteamAuthenticator/App.xaml.cs

```diff
@@ -14,9 +14,9 @@ namespace UnofficialSteamAuthenticator
     /// <summary>
     /// Provides application-specific behavior to supplement the default Application class.
     /// </summary>
-    public sealed partial class App : Application
```

Comment:
It's a partial class that has some properties from somewhere else, but where?
Search the filesystem to find out!